cogs5e/sheetManager.py

```diff
@@ -488,13 +488,14 @@ async def character_channel(self, ctx):
 
         await try_delete(ctx.message)
 
-    @character.command(name="reset_all")
+    @character.command(name="resetall")
     @commands.guild_only()
-    async def reset_all(self, ctx):
+    async def resetall(self, ctx):
```

Comment:
Use the snake case name for the function, only change the command name with the decorator (as you have done)